src/js/modules/articles/components/ArticlePage.js

```diff
@@ -83,30 +117,30 @@ const ArticlePage = ({
             href="https://issuu.com/stuyspectator/docs"
             target="_blank"
           >
-            {`Volume ${toRoman(article.volume)}, Issue ${article.issue}`}
+            {`Volume ${toRoman(
+              articleBySlug.volume,
+            )}, Issue ${articleBySlug.issue}`}
           </a>
           .
         </Col>
         <Col xsHidden smHidden md={3} lg={3} />
       </Row>
       <RecommendedRow
-        section={section.parentId ? sections[section.parentId] : section}
+        section={section.parent_section ? section.parent_section : section}
       />
-      <CommentThread article={article} />
+      <CommentThread article={articleBySlug} />
```

Comment:
I'm curious why you're passing the article to everything. There's probably a good reason, but I'd like to know why

Reply:
Article is needed in the `CommentThread` because, to post a comment, you need the articleId. UPDATE: this has been changed to articleId prop.